input/metadata/ARL_metadata.json

```
@@ -138,10 +152,21 @@
         "minimum": 0,
         "default": 42
       },
+      "set_seed": {
+        "type": "bool",
+        "description": "Whether or not to set random seed",
+        "default": true
+      },
       "simulate_animals": {
         "type": "bool",
         "description": "Whether or not to simulate animals during the simulation",
         "default": true
+      },
+      "nutrient_standard": {
+        "type": "string",
+        "default": "NASEM",
+        "pattern": "(NASEM|NRC)",
```

```suggestion
        "pattern": "^(NASEM|NRC)$",
```
Might be a little overkill, but is what `"energy_and_nutrient_calculation_method"` had.